src/python/pants/engine/subsystem/native.py

```diff
@@ -166,7 +168,7 @@
     ExecutionStat execution_execute(RawScheduler*);
     RawNodes* execution_roots(RawScheduler*);
 
-    void validator_run(RawScheduler*, TypeId*, uint64_t);
+    Value* validator_run(RawScheduler*, TypeId*, uint64_t);
```

Comment:
Makes sense. How do I ensure python drops the handle? Do I need to adjust Storage?